whois-commons/src/test/java/net/ripe/db/whois/common/dao/jdbc/DatabaseHelper.java

```diff
@@ -524,13 +524,13 @@ public Object doInStatement(Statement statement) throws SQLException, DataAccess
 
     public void deleteAuthoritativeResource(final String source, final String resource) {
         internalsTemplate.execute("delete from authoritative_resource where source ='"+source+"' and resource = '"+resource+"'");
-        authoritativeResourceData.refreshAuthoritativeResourceCacheOnChange();
+        authoritativeResourceData.refreshActiveSource();
```

Comment:
Does this *always* refresh the cache, rather than refreshing on change? Does this change the test behaviour?

Reply:
Yes it does, I've removed all the code where there was a lot of fiddling going on with the timestamp to get the job to refresh the authoritative resources so it's actually a lot simpler and more straightforward to use atm (as in telling the authoritative cache to refresh now actually always refreshes the cache).